release_notes.md

```
@@ -0,0 +1,79 @@
+ - Linked Issues:
+   - Implement Create Tour Page: https://github.com/bluewave-labs/bluewave-onboarding/issues/213
+
+## did fade in for default page
+ - Created by: erenfn
+ - PR Link: https://github.com/bluewave-labs/bluewave-onboarding/pull/209
+ - Merged on: 2024-09-23
+ - Linked Issues:
+   - Filler Page is shown while we wait for fetching banners/popups in default page: https://github.com/bluewave-labs/bluewave-onboarding/issues/207
+
+## Update Home.jsx
+ - Created by: erenfn
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)